lib/Extension/LanguageServerPsalm/LanguageServerPsalmExtension.php

```diff
@@ -85,13 +89,15 @@ public function configure(Resolver $schema): void
         ]);
         $schema->setTypes([
             self::PARAM_PSALM_BIN => 'string',
+            self::PARAM_PSALM_CONFIG => 'string',
```

Comment:
maybe remove this, I don't think we support union types - i.e. can't do `null|string`